### ISSUE TITLE
Support air sensor for monitoring PM2.5 and PM10

### ISSUE THREAD

**Issue by hekonsek** [rhiot/rhiot#302]
It would be great to monitor PM2.5 and PM10 pollution using sensor like this [1].

[1] http://botland.com.pl/czujniki-gazu/4480-czujnik-pylu-stezenia-czastek-monitor-czystosci-powietrza-gp2y1010au0f.html

**Comment by hekonsek** [comment]
@gautric We've been talking with @ajurasz about PM2.5 and PM10 monitoring. We are thinkin gabout buying this [1] sensor. Do you think it will be suitable for a I2C driver?

[1] http://botland.com.pl/czujniki-gazu/4480-czujnik-pylu-stezenia-czastek-monitor-czystosci-powietrza-gp2y1010au0f.html

**Comment by ppatierno** [comment]
Sorry @hekonsek but I started to follow this interesting project few days ago. Great work !
I'll setup it in the next days :-) 

Pay attention that the sensor isn't based on I2C protocol. I see that it has only an analog output for the microcontroller so you need an ADC (outside or inside the microcontroller if available).
The discrete value must be converted in dust density using the dust density characteristic in the datasheet.

**Comment by gautric** [comment]
I am not fluent in Polish but I think "Wyjście analogowe: 0,5 V / (100 μg/m3)." mean analog output.
I didn't found any i2c protocol into documentation
but the good news, camel-pi4j w/ analog input should work out-of-the-box ;-)

**Comment by hekonsek** [comment]
@ppatierno Looking forward for our cooperation! Ping me if you need anything :) . Good point with that analog only stuff.

@gautric  Do we have any example of using camel-pi4j with analog input? I'm trying to comprehend what is required to make it work.

**Comment by ppatierno** [comment]
Pay attention that both Pi boards (1 and 2) don't have analog inputs out of the box ... you have to use an external ADC.

@hekonsek thanks !

**Comment by ajurasz** [comment]
@hekonsek from what I read @ppatierno is right about Pi boards missing analog inputs and this can be   easily handled as described here [1]. Then we should be able to use camel-pi4j.

[1] https://learn.adafruit.com/reading-a-analog-in-and-controlling-audio-volume-with-the-raspberry-pi

**Comment by ppatierno** [comment]
@ajurasz yes as I said an external ADC is needed and the article you refer is a good example
